hack/examples/bases/nutanix/calico/crs/kustomization.yaml.tmpl

```diff
@@ -0,0 +1,27 @@
+# Copyright 2024 D2iQ, Inc. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
```

Comment:
Lets start with only the requirement which is Cilium and then we can add Calico support in the future.
(I'm trying to be mindful of the size of the PR and the test matrix)

Reply:
we will have to modify sync.sh to ignore non existent folders. keeping it short thru https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/pull/473